ironfish/src/rpc/response.ts

```diff
@@ -34,6 +34,12 @@ export class RpcResponse<TEnd = unknown, TStream = unknown> {
     timeout: SetTimeoutToken | null,
   ) {
     this.promise = promise
+    this.promise.catch(() => {
+      // Eat the exception so there isn't an unhandled exception
+      // error that crashes the process. You can still read/catch
+      // the exception using waitForEnd().catch() or try..catch
+      // around contentStream() for non-RpcConnectionLostError errors
+    })
```

Comment:
Are we sure we want this exception to be eaten for all requests? If you start a request and don't wait for it to end do we not want those to bubble as an unhandled rejection?